pertpy/tools/_dialogue.py

```diff
@@ -82,27 +82,27 @@ def _get_pseudobulks(
 
         return pseudobulk
 
-    def _pseudobulk_pca(self, adata: AnnData, groupby: str, n_components: int = 50) -> pd.DataFrame:
-        """Return cell-averaged PCA components.
+    def _pseudobulk_feature(
```

Comment:
```suggestion
    def _pseudobulk_space(
```

Maybe more something like this?